README.md

```diff
@@ -9,6 +9,36 @@ This repository contains the Strimzi canary tool implementation.
 It acts as an indicator of whether Kafka clusters are operating correctly.
 This is achieved by creating a canary topic and periodically producing and consuming events on the topic and getting metrics out of these exchanges.
 
+## Deployment
+
+In order to deploy the Strimzi canary in the Kubernetes cluster alongside the already running Apache Kafka cluster, first download the installation files from one of the [releases](https://github.com/strimzi/strimzi-canary/releases) you are interested in.
+Unzip the release package and edit the `Deployment` in order to specify the right bootstrap servers for connecting to the Apache Kafka cluster; it is defined by the `KAFKA_BOOTSTRAP_SERVERS` environment variable.
+The provided `Deployment` has also TLS disabled by default (`TLS_ENABLED` as `false`) and the canary producing and consuming messages every 10 seconds (`RECONCILE_INTERVAL_MS` as `"10000"`).
+
+Then just run the following command:
+
+```shell
+kubectl apply -f ./install
+```
+
+Other than creating the corresponding `Deployment`, the canary will run with a specific `ServiceAccount` and a `Service` will be created in order to make the Prometheus metrics accessible via HTTP on port `8080`.
+
+### Encryption and TLS
+
+If the Apache Kafka cluster has the TLS protocol enabled on the listener where the canary should connect to in order to encrypt the traffic, enable the TLS configuration in the canary `Deployment` using the corresponding environment variables.
+
+The `TLS_ENABLED` has to be set as `true` and the `TLS_CA_CERT` has to contain the cluster CA certificate, in PEM format, used to sign the broker certificates; if left empty, the canary will use the system certificates.
+Leaving the Strimzi Cluster Operator genberating the cluster CA certificate, it can be extracted from the corresponding `<cluster_name>-cluster-ca-cert` `Secret`.
```

Comment:
```suggestion
Set `TLS_ENABLED` to `true` in the `Deployment` file. You'll also need a cluster CA certificate in PEM format that validates the identity of the Kafka brokers. You can reference a cluster CA certificate using the  `TLS_CA_CERT` environment variable. If you leave `TLS_CA_CERT` empty, canary can use the system certificates generated by the Cluster Operator. You can extract the cluster CA certificate from the `<cluster_name>-cluster-ca-cert` `Secret`.
```

Reply:
Might not have read the original draft correctly, but I'm assuming you have to reference a certificate whether it's your own or the system cert

Reply:
No it's not this way. You can specify the cluster CA that could be generated by Strimzi or the one provided by the user (but anyway I think these are too many details). If you leave the env var empty, you are relying on "system certificates" that are the ones for well known CAs and are typically already installed on any laptop we use (see for example Verisign, Let's Encrypt and so on).

Reply:
I will change it from your suggestion.